### PR TITLE
Added ex.1 insight panel bundle docs

### PR DESCRIPTION
**Pull request by phazonoverload**
This PR introduces 4 new sections in the app/insights page:

1. Bar chart
2. Meter
3. Line chart
4. Pie/donut chart

I need a general review but would like specific focus in two areas:

1. Technical correctness. I learnt about these today and wrote this page fairly swiftly. I believe I have got things correct by primarily combining existing phrasing and only writing new panel-specific options.
2. The phrasing around in what context these panels are available.

### CODE REVIEW

File: app/insights.md [hunk 1]
```diff
@@ -257,3 +257,101 @@ Your variable's data type will need to be a valid match to wherever you are usin
...
+- **Group Aggregation** – Selects the type of aggregation to perform. If you may only select a Field to be presented on
+  the x=axis [Learn more](#more-on-aggregate-functions).
+- **Function** – Selects the type of aggregate to perform.
+- **Filter** – Allows filtering of Items considered for analysis based on logical conditions.
+- **Value Decimals** – Changes the number of decimals displayed on the both axes.
+- **Color** – Sets the default color of your line.
+- **Show Axis Labels** - Selects if and on which axis labels are shown.
+- **Show Tooltip Marker** - If enabled, shows data when a data point is hovered over.
+- **Curve Type** - Selects the style of the curve - smooth, straight, or stepped.
+- **Conditional Styles** – Changes color of bar if its value is `>`, `>=`, `<`, `<=`, `==`, or `!=` to some defined
```
Comment (azrikahar) [inline app/insights.md:308]:
```suggestion
- **Conditional Styles** – Changes color of line if its value is `>`, `>=`, `<`, `<=`, `==`, or `!=` to some defined
```

**Review by azrikahar — CHANGES_REQUESTED** [review]
> The phrasing around in what context these panels are available.

In regards to this, I don't think there's any issue with it, but maybe we should align it with https://docs.directus.io/app/layouts.html#kanban-layout-cloud-exclusive. 

Currently Kanban Layout is only appending `[Cloud Exclusive]` at the end (and IIRC this term might be outdated since it's slightly excluding Enterprise self-hosted), so we may need to update Kanban Layout to use the same Availability info notice for consistency.

In case we're pivoting to something such as "Exclusive Extensions" (I think this was brought up recently), perhaps we can also use the badge approach: https://vitepress.vuejs.org/guide/theme-badge

File: app/insights.md [hunk 2]
```diff
@@ -257,3 +257,101 @@ Your variable's data type will need to be a valid match to wherever you are usin
...
+
+- **Collection** – Selects the Collection you want to analyze.
+- **Field** – Selects the Field to run aggregate function on.
+- **Function** – Selects the type of aggregate to perform.
+- **Donut** - If enabled, a circle is cut out of the center of the chart.
+- **Show Labels** - If enabled, the percentage value is shown in each segment.
+- **Show Legend** - Selects if and where the legend is shown - none, right, or bottom.
+- **Filter** – Allows filtering of Items considered for analysis based on logical conditions.
+- **Value Decimals** – Changes the number of decimals displayed in the chart segments.
+- **Color** – Sets the default base color of your dial. Other segments will be darkened variations.
```
[the remark this answers is not in the document]
Comment (phazonoverload) [reply app/insights.md:355]: Yup this was a copy/paste typo from meter.

File: app/insights.md [hunk 3]
```diff
@@ -257,3 +257,101 @@ Your variable's data type will need to be a valid match to wherever you are usin
...
+:::
+
+The Bar Chart Panel allows you to build line charts from data held in collections, and are often used for time-series
+data.
+
+- **Collection** – Selects the Collection you want to analyze.
+- **X-Axis** - The Field presented on the x-axis.
+- **Y-Axis** - The Field presented on the y-axis.
+- **Group Aggregation** – Selects the type of aggregation to perform. If you may only select a Field to be presented on
+  the x=axis [Learn more](#more-on-aggregate-functions).
```
[the remark this answers is not in the document]
Comment (phazonoverload) [reply app/insights.md:300]: This was a copy/paste from somewhere else on this page, so I'll fix it there too

**Comment by phazonoverload** [comment]
Agree with use of badge, but not for today.

**Review by azrikahar — APPROVED** [review]
LGTM! Nice catch on updating the Cloud Exclusives glossary as well 👍

**Pull request merged**